common/src/View/QtUtils.h

```diff
@@ -101,6 +101,11 @@ namespace TrenchBroom {
             window->restoreState(settings.value(path).toByteArray());
         }
 
+        /**
+         * Return true if the given widget or any of its children currently has focus.
+         */
+        bool hasTransitiveFocus(const QWidget* widget);
```

Comment:
Naming nitpick, I interpret transitive to mean one of the parents has focus, but this is the opposite. Maybe "widgetOrChildHasFocus"?

Reply:
Renamed in c603485906623754ff554c6499ec1c50303c747f.